### PR TITLE
formik/ENG-5162 implement widget forms with formik

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

File: src/ui/widget-forms/publish-single-content-config/SingleContentConfigFormBody.js [hunk 1]
```diff
@@ -267,7 +266,7 @@ export class SingleContentConfigFormBody extends PureComponent {
         </div>
         <ContentsFilterModalContainer
           modalTitleText={intl.formatMessage({ id: 'cms.contents.modal.filter.title' })}
-          invalid={!isValid}
+          invalid={false}
```
Comment (ryanjpburgos) [inline src/ui/widget-forms/publish-single-content-config/SingleContentConfigFormBody.js:269]: is it correct to have always false?

File: src/ui/widgets/common/WidgetForm.js [hunk 2]
```diff
@@ -25,6 +25,15 @@ export const MODE_CLONE = 'clone';
...
 
+export const validateFieldValueForFormikValidators = (value, validators) => {
+  let error = null;
+  validators.forEach((validator) => {
+    const validationError = validator(value);
+    if (validationError) error = validationError;
+  });
+  return error;
+};
+
```
Comment (ryanjpburgos) [inline src/ui/widgets/common/WidgetForm.js:36]: here you can replace the function with the utility function that we have in formikUtils

File: src/ui/widgets/common/WidgetForm.js [hunk 3]
```diff
@@ -25,6 +25,15 @@ export const MODE_CLONE = 'clone';
 const maxLength30 = maxLength(30);
 const maxLength70 = maxLength(70);
 
+export const validateFieldValueForFormikValidators = (value, validators) => {
+  let error = null;
+  validators.forEach((validator) => {
+    const validationError = validator(value);
+    if (validationError) error = validationError;
+  });
+  return error;
```
Comment (ryanjpburgos) [inline src/ui/widgets/common/WidgetForm.js]: here you can replace the function with the utility function that we have in formikUtils

**Review by ryanjpburgos — CHANGES_REQUESTED** [review]

File: src/ui/widget-forms/publish-single-content-config/SingleContentConfigFormBody.js (hunk 1 above)
Comment (ichalagashvili) [reply src/ui/widget-forms/publish-single-content-config/SingleContentConfigFormBody.js:269]: yes it is intended like that. Good catch!

**Review by ryanjpburgos — APPROVED** [review]

**Pull request merged**